### PR TITLE
Add tentree ads

### PR DESCRIPTION
**Pull request by amtims**

### CODE REVIEW

File: Extension/response.js [hunk 1]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+        "weight": 0.8
+      },
+      {
+        "tag": "mens",
+        "weight": 0.4
+      },
+      {
+        "tag": "pants",
+        "weight": 0.8
+      }
```
Comment (CubeOfCheese) [inline Extension/response.js:1049]: Probably should have slightly lower weight than "sweats"

File: Extension/response.js [hunk 2]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+        "weight": 0.4
+      },
+      {
+        "tag": "mens",
+        "weight": 0.4
+      },
+      {
+        "tag": "button up",
+        "weight": 0.4
+      }
```
Comment (CubeOfCheese) [inline Extension/response.js:1084]: currently we can't use multi-word product tags. Maybe just use the word button or add a hyphen or both

File: Extension/response.js [hunk 3]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+        "weight": 0.4
+      },
+      {
+        "tag": "tree",
+        "weight": 0.4
+      },
+      {
+        "tag": "ev2",
+        "weight": 0.4
+      }
```
Comment (CubeOfCheese) [inline Extension/response.js:1236]: What does ev2 mean? is this something that will be likely to appear on other shop sites? if not, then it shouldn't be included

File: Extension/response.js [hunk 4]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+        "weight": 0.8
+      },
+      {
+        "tag": "covid",
+        "weight": 0.8
+      },
+      {
+        "tag": "3-pack",
+        "weight": 0.1
+      },
```
Comment (CubeOfCheese) [inline Extension/response.js:1434]: I don't think this helps with relevancy

File: Extension/response.js [hunk 5]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+  },
+{
+    "productName":"Travel Cubes",
+    "website":"https://shareasale.com/r.cfm?b=1240004&u=2552760&m=81987&urllink=&afftrack=",
+    "companyName": "Tentree",
+    "companyScore": 9.3,
+    "price":30.80,
+    "productTags": [
+      {
+        "tag": "travel cubes",
```
Comment (CubeOfCheese) [inline Extension/response.js:1668]: break this up into two tags

File: Extension/response.js [hunk 6]
```diff
@@ -984,5 +984,820 @@ export let ads = [
...
+    "price":32.00,
+    "productTags": [
+      {
+        "tag": "beanie",
+        "weight": 0.8
+      },
+      {
+        "tag": "hat",
+        "weight": 0.8
+      },
```
Comment (CubeOfCheese) [inline Extension/response.js:1732]: hat shouldn't have equal relevance as beanie since beanie is more specific

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]
Throughout, there are the following problems:
multiword tags
tags that are a word from the title that as far as I can tell are random names and not actually descriptive of the product

I'm excited about all these new products!

File: Extension/response.js (hunk 3 above)
Comment (amtims) [reply Extension/response.js]: I'm not sure it was just in the title so I can delete it

**Comment by amtims** [comment]
Okay I just edited for changes let me know if it shows up.

**Review by CubeOfCheese — APPROVED** [review]
Sweet stuff dude. Hopefully we can get this update out soon

**Pull request merged**